### PR TITLE
feat: migrate to itk-wasm

### PR DESCRIPTION
**Pull request by cjh1**
This also moves the storage of dataset files to the main thread.

### CODE REVIEW

File: src/store/datasets-dicom.ts [hunk 1]
```diff
@@ -117,85 +118,104 @@ export const useDICOMStore = defineStore('dicom', {
...
 
-            this._updateDatabase(patient, study, volumeInfo);
-          }
-
-          // invalidate any existing volume
-          if (volumeKey in this.volumeToImageID) {
-            // buildVolume requestor uses this as a rebuild hint
-            set(this.needsRebuild, volumeKey, true);
+            // Add file entries to the file store
+            fileStore.add(volumeKey, volumeFiles);
```
Comment (floryst) [inline src/store/datasets-dicom.ts:146]: In `deleteVolume`, we want to have the corresponding remove operation in the fileStore.

**Review by floryst — COMMENTED** [review]
Looking good so far!

File: src/store/datasets-dicom.ts (hunk 1 above)
Comment (cjh1) [reply src/store/datasets-dicom.ts:146]: That is done [here](https://github.com/cjh1/paraview-medical/blob/dicom-db/src/store/datasets.ts#L259).

Comment (floryst) [reply src/store/datasets-dicom.ts:146]: Oh, I see now. Thanks!

**Comment by floryst** [comment]
LGTM!

**Pull request merged**